src/components/HomePage.js

```diff
@@ -0,0 +1,64 @@
+// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
- [ ] Please avoid disabling linters as they are very useful to keep a consistent code format and ensure the use of best practices. Instead, please consider solving the errors! 🤓 🔧

Reply:
Can be improved!